### PR TITLE
Bug/cant scroll user highscores

### PR DESCRIPTION
**Pull request by DominikGuzei**
Makes the user highscores super awesome scrollable with iScroll which works in all modern browsers + on mobile platforms. It looks like the native iPad scrollbars in the browser.

### CODE REVIEW

File: app/assets/javascripts/helpers/ember/view_in_dom_flag.js.coffee [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Ember.View.reopen
+  didInsertElement: ->
+    @set('elementInDOM', true)
+    @_super()
+
+  willDestroyElement: ->
+    @set('elementInDOM', false)
+    @_super()
```
Comment (tombu) [inline app/assets/javascripts/helpers/ember/view_in_dom_flag.js.coffee:8]: I don´t think we need that. Each `Ember.View` has a property `state` which returns `inDOM` if the view is in the DOM.

Comment (DominikGuzei) [inline app/assets/javascripts/helpers/ember/view_in_dom_flag.js.coffee:8]: which is internal and should not be used by code ;-) I looked it up and found this solution on stack overflow

Comment (tombu) [inline app/assets/javascripts/helpers/ember/view_in_dom_flag.js.coffee:8]: ok .. ^^

**Comment by stravid** [comment]
Can be merged in my opinion!

**Pull request merged**